app/build.gradle

```diff
@@ -71,6 +72,9 @@ dependencies {
 
     implementation 'com.google.android.gms:play-services-maps:17.0.0'
 
+    implementation 'com.android.support:recyclerview-v7:30'
+
+
```

Comment:
Why add 2 spaces here ? I think it would be more consistent to only have one

Reply:
spaces were due to the merging process